cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -650,6 +754,7 @@ func resetProxyEnv(t *testing.T) map[string]string {
 	t.Log("Saved environment: ", savedEnv)
 
 	os.Setenv("HTTP_PROXY", "http://proxy.example.com:3128")
+	os.Setenv("HTTPS_PROXY", "http://proxy.example.com:3128")
```

Comment:
Before I delete the above two UTs, TestHTTPProxyCheck and TestHTTPProxyCIDRCheck will be skipped.
When deleted, these two UTs will fail. This line solves the problem

Reply:
i'm not sure i understand why this is failing, but adding it seems fine.

should the value be `"https://proxy.example.com:3128"` (https instead of http)?